src/dashboard/src/components/transfer/views.py

```diff
@@ -234,7 +231,7 @@ def cleanup_metadata_set(request, set_uuid):
         response["deleted_objects"] = len(objects)
         objects.delete()
         models.TransferMetadataSet.objects.get(id=set_uuid).delete()
-    except Exception as e:
-        response["message"] = e.message
+    except Exception as err:
+        response["message"] = "{}".format(err)
```

Comment:
Being a bit pedantic here, if we didn't want to use `format` would the following be more correct?

```suggestion
        response["message"] = six.text_type(err)
```